blog.md

```diff
@@ -74,4 +114,28 @@ return new FileStreamResult(stream, System.Net.Mime.MediaTypeNames.Application.Z
 {
     FileDownloadName = "remeshed.zip"
 };
-```
\ No newline at end of file
+```
+
+## Testing
+
+So, we have shown how to build and deploy two example nodes for a 3D content pipeline but in order to test this I currently need to carry out the following:
+
+- Take the model and textures from [here](https://github.com/peted70/az-func-blender/tree/master/TestModels) and add all contents of that folder to a zip archive as it is organized in the folder

```

Comment:
Are these steps in order? If yes, I think using numbers would be a better choice.

Reply:
`1. will always create a number appropriate to the context (either the next number or 1.)`

Reply:
Agree and updated